Formula/ahoy.rb

```diff
@@ -20,13 +20,7 @@ class Ahoy < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    bin_path = buildpath/"src/github.com/ahoy-cli/ahoy"
-    bin_path.install Dir["*"]
-    cd bin_path do
-      system "go", "build", "-o", bin/"ahoy", "-ldflags", "-X main.version=#{version}", "."
-    end
+    system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}-homebrew", "-o", bin/"ahoy", "."
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w X main.version=#{version}-homebrew")
```

Reply:
Thanks for this suggestion @chenrui333 - I played around with the `std_go_args()` but couldn't get them to put my version string in properly, but I'll try again now and hopefully can get it working.

Reply:
interesting

Reply:
it builds/tests out fine in my local.

Reply:
Got this working now, thanks 👍